sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcSchemaIOProvider.java

```diff
@@ -65,6 +65,7 @@ public Schema configurationSchema() {
         .addNullableField("readQuery", FieldType.STRING)
         .addNullableField("writeStatement", FieldType.STRING)
         .addNullableField("fetchSize", FieldType.INT16)
+        .addNullableField("disableAutoCommit", FieldType.BOOLEAN)
```

Comment:
from the history this is likely to break python jdbcIO (based on schemaIO), see #26480

Consider either revert the change of JdbcSchemaIOProvider.java or add the same field to jdbc.py and same order

Reply:
good catch. added to jdbc.py